### ISSUE TITLE
VTT Dice roller

### ISSUE THREAD

**Issue by Lambosaurus** [Lambosaurus/caltrops#15]
A UI element should ask the user to enter the following items
 * An Aspect
 * A Skill
 * Bonus dice (may be negative)

I do not yet have a solid proposal for the UI. Perhaps simply a button in the header and a modal?

I like the idea of clicking on the table rows to have them selected, to prevent the user needing to select them in a modal.
However, this may interfere with #6 

Upon entering them, this information along with the dice total should be forwarded to foundry VTT.

**Comment by Lambosaurus** [comment]
As a UI suggestion: a dice symbol could be added to each skill.
Selecting this would open a modal asking for an Aspect and an optional bonus.

**Comment by Lambosaurus** [comment]
Development notes:

Using https://github.com/kakaroto/fvtt-module-api, I can submit foundry requests without bothering about websocket operations. This webapp could function without dependency on this module - the websocket component would need to be debugged and replicated.

I get a successful roll using the following arguments:
```
// These are submitted as query parameters, and need to be first URLEncoded.
name="modifyDocument"
arg0= {
    "type": "ChatMessage",
    "action": "create",
    "data": [
        {
            "_id": null,
            "type": 5,
            "user": "O7kFbsMVJmbULJM0",
            "timestamp": 1664614795053,
            "content": "14",
            "speaker": {
                "scene": "b9HRghnNyn5pb3zg",
                "actor": null,
                "token": null,
                "alias": "tim"
            },
            "whisper": [],
            "blind": false,
            "roll": "{\"class\":\"Roll\",\"options\":{},\"dice\":[],\"formula\":\"1d20\",\"terms\":[{\"class\":\"Die\",\"options\":{},\"evaluated\":true,\"number\":1,\"faces\":20,\"modifiers\":[],\"results\":[{\"result\":14,\"active\":true}]}],\"total\":14,\"evaluated\":true}",
            "sound": "sounds/dice.wav",
            "emote": false,
            "flags": {}
        }
    ],
    "options": {
        "temporary": false,
        "renderSheet": false,
        "render": true,
        "rollMode": "roll"
    }
}
```

This leaves two key tasks:
1. The arguments were found by debugging a `/r 1d20` command in foundry. The body needs to be generated correctly.
2. The websocket component and login process needs to be replicated.

**Comment by Lambosaurus** [comment]
Resolved here: 60d969f24a4dcab5874b3c420c26f6e2e551b754
